packages.md

```diff
@@ -4,7 +4,40 @@ This page contains packages recommended by HelloFresh Golang Clan. These are not
 
 If you need more packages or just for inspiration, visit [Awesome Go](https://awesome-go.com/).
 
+## Configuration
+
+- [github.com/kelseyhightower/envconfig](https://github.com/kelseyhightower/envconfig) - get configs from environment variables
+
```

Comment:
Please add `github.com/spf13/viper` here - normally we use this in our projects and `github.com/kelseyhightower/envconfig` is used by viper for environment backend.